cartoframes/data/clients/bigquery_client.py

```diff
@@ -33,11 +55,13 @@ def upload_dataframe(self, dataframe, schema, tablename, project, dataset, ttl_d
             table.expires = expiration
             self.client.update_table(table, ["expires"])
 
+    @refresh_client
     def query(self, query, **kwargs):
         response = self.client.query(query, **kwargs)
 
         return response
 
+    @refresh_client
     def delete_table(self, tablename, project, dataset):
```

Comment:
User tokens are not granted to delete tables. Instead of that we added a TTL for tables.

Please consider removing this method.

cc. @alejandrohall

Reply:
Ok, @alejandrohall is working on a refactor of the enrichment methods. Since this one applies there, we will tackle it there